backend/src/database/populationUtils/sessionPopulation.ts

```diff
@@ -0,0 +1,12 @@
+import { addSession } from '../collectionsUtils/sessionUtils'
+import faker from 'faker';
+
+export const populateSessionsCollection = async (n = 10, role = "student") => {
```

Comment:
Po co tu jest role?

Reply:
Idk
Ale nie przeszkadza :D 
A tak serio to usunę. Zostało z ctrl+c; ctrl+v